presto-main/src/test/java/com/facebook/presto/sql/gen/TestExpressionCompiler.java

```diff
@@ -129,6 +131,13 @@
     private FunctionAssertions functionAssertions;
     private List<ListenableFuture<Void>> futures;
 
+    private static <T> T[] concat(T[] first, T[] second)
```

Comment:
Thanks, somehow I missed that.

This can use `ObjectArrays.concat()` from Guava (it is slightly different in that it requires a `Class`).